b2sdk/bucket.py

```diff
@@ -232,7 +232,7 @@ def download_file_by_id(self, file_id, download_dest, progress_listener=None, ra
           download_file_by_id actually belongs in :py:class:`b2sdk.v1.B2Api`, not in :py:class:`b2sdk.v1.Bucket`; we just provide a convenient redirect here
 
         :param str file_id: a file ID
-        :param str download_dest: a local file path
+        :param b2sdk.v1.DownloadDestLocalFile download_dest: a download destination object to use
```

Comment:
can this really only be DownloadDestLocalFile and not any other class? I think there is an interface for it, see one line below